lib/jdomodels/src/main/java/org/sagebionetworks/repo/throttle/CountingSemaphoreThrottle.java

```diff
@@ -22,16 +24,16 @@ public class CountingSemaphoreThrottle {
 
 	@Around("execution(* org.sagebionetworks.database.semaphore.CountingSemaphoreImpl.*(..))")
 	public Object profile(ProceedingJoinPoint pjp) throws Throwable {
-		Object result = null;
+		Optional<String> result = null;
```

Comment:
It seems that all of the other methods work because they are void, so the result is still null, so the cast does not fail.  I will change the cut point to target the method.